frontend/class-frontend.php

```diff
@@ -1732,8 +1732,21 @@ function embed_rssfooter_excerpt( $content ) {
 	 * @return string
 	 */
 	function embed_rss( $content, $context = 'full' ) {
-		if ( is_feed() ) {
 
+		/**
+		* Dynamically include/exclude the RSS footer
+		*
+		* Allow other plugins to show/hide the RSS footer dynamically - useful for non-standard RSS feeds
+		*
+		* @since 2.1
+		*
+		* @param boolean $show_embed Indicates if the embed should be shown or not
```

Comment:
Just noticed this documentation isn't following the WP SEO standard for documenting filters :)

A good example [can be found here](https://github.com/Yoast/wordpress-seo/blob/trunk/frontend/class-frontend.php#L977).